CMakeLists.txt

```diff
@@ -127,16 +128,23 @@ set(CLP_FFI_PY_LIB_IR_SOURCES
     ${CLP_FFI_PY_LIB_SRC_DIR}/PyExceptionContext.hpp
     ${CLP_FFI_PY_LIB_SRC_DIR}/PyObjectCast.hpp
     ${CLP_FFI_PY_LIB_SRC_DIR}/PyObjectUtils.hpp
-    ${CLP_FFI_PY_LIB_SRC_DIR}/Python.hpp
     ${CLP_FFI_PY_LIB_SRC_DIR}/utils.cpp
     ${CLP_FFI_PY_LIB_SRC_DIR}/utils.hpp
 )
 
+# NOTE: these headers are wrappers of third-party dependency libraries. They are required to silent
+# clang-tidy warnings for missing direct includes.
```

Comment:
* Let's ensure we always use a capital after colons, unless the next word must be lowercase (e.g., a variable name).
* "silent" is the adjective, "silence" is the verb.

Reply:
Sorry for the missing capitalization